frontend/src/metabase/ui/components/overlays/Modal/Modal.styled.tsx

```diff
@@ -24,6 +24,9 @@ export const getModalOverrides = (): MantineThemeOverride["components"] => ({
       content: {
         backgroundColor: "var(--mb-color-background)",
       },
+      header: {
+        backgroundColor: "var(--mb-color-background)",
+      },
```

Comment:
In dark-themed apps, the header looks ugly without this.

![CleanShot 2567-11-20 at 00 34 21@2x](https://github.com/user-attachments/assets/5bda63b2-16d2-43b7-b2df-a0c906748053)